src/ui/ResultsPerPage/ResultsPerPage.ts

```diff
@@ -13,6 +13,8 @@ import { exportGlobally } from '../../GlobalExports';
 import { l } from '../../strings/Strings';
```

Comment:
In order to minimize the number of abstractions, I simply copy-pasted the code in Pager into ResultsPerPage. Same for the unit tests :/. That said, what do you think of adding a class like `PageComponentsManager` that components could register to? It would be in charge of listening and setting the visibility.